Description: in Python 3, "map()" returns a map object, not a list, making the
test fail with "TypeError: can only concatenate list (not "map") to list".
Also, Python 3 doesn't have "basestring", which was making another test fail.
Forwarded: https://github.com/joeyespo/path-and-address/pull/2
Author: Tiago Ilieve <tiago.ilieve@gmail.com>

--- a/path_and_address/tests/test_parsing.py
+++ b/path_and_address/tests/test_parsing.py
@@ -1,6 +1,9 @@
 from itertools import product
+import sys
 from ..parsing import resolve, split_address
 
+if sys.version_info[0] >= 3:
+    basestring = str
 
 paths = [
     '0.0.0.0',
--- a/path_and_address/tests/test_validation.py
+++ b/path_and_address/tests/test_validation.py
@@ -7,7 +7,7 @@
 
 
 def _join_all(hostnames, ports):
-    return map(_join, product(hostnames, ports))
+    return list(map(_join, product(hostnames, ports)))
 
 
 hostnames = [
